rest/rest-sqs/src/main/scala/org/elasticmq/rest/stats/StatisticsRestServerBuilder.scala

```diff
@@ -0,0 +1,159 @@
+package org.elasticmq.rest.stats
+
+import java.util.concurrent.TimeUnit
+import java.util.concurrent.atomic.AtomicReference
+
+import akka.actor.{ActorRef, ActorSystem, Props}
+import akka.http.scaladsl.Http
+import akka.stream.ActorMaterializer
+import akka.util.Timeout
+import org.elasticmq._
+import org.elasticmq.actor.QueueManagerActor
+import org.elasticmq.rest.sqs.QueueAttributesOps
+import org.elasticmq.rest.sqs.directives.ElasticMQDirectives
+import org.elasticmq.util.{Logging, NowProvider}
+
+import scala.concurrent.duration._
+import scala.concurrent.{Await, Future}
+import scala.util.control.NonFatal
+
+case class TheStatisticsRestServerBuilder(
+                                           providedActorSystem: ActorSystem,
+                                           providedQueueManagerActor: Option[ActorRef],
```

Comment:
maybe it's possible to remove this option as well?

Reply:
👍